cellfinder_napari/tests/data/brainreg-napari-output/brainreg.json

```diff
@@ -0,0 +1 @@
+{"image_paths": null, "backend": "niftyreg", "voxel_sizes": ["50.0", "40.0", "40.0"], "orientation": "psl", "data_orientation": "psl", "atlas": "example_mouse_100um", "atlas_key": "atlas_key.example_mouse_100um", "registration_output_folder": "/Users/dstansby/brainreg-napari-output", "affine_n_steps": "6", "affine_use_n_steps": "5", "freeform_n_steps": "6", "freeform_use_n_steps": "4", "bending_energy_weight": "0.95", "grid_spacing": "10", "smoothing_sigma_reference": "1", "smoothing_sigma_floating": "1.0", "histogram_n_bins_floating": "128.0", "histogram_n_bins_reference": "128.0"}
```

Comment:
Locally tests fail the assertion on Line 57 of `test_transform.py`

Manually emulating the test in the GUI gives:
```python
Traceback (most recent call last):
  File "src\psygnal\_signal.py", line 950, in _run_emit_loop
  File "src\psygnal\_weak_callback.py", line 262, in cb
  File "C:\Users\Alessandro\anaconda3\envs\brainglobe\lib\site-packages\magicgui\widgets\_function_gui.py", line 218, in _disable_button_and_call
    self.__call__()
  File "C:\Users\Alessandro\anaconda3\envs\brainglobe\lib\site-packages\magicgui\widgets\_function_gui.py", line 331, in __call__
    value = self._function(*bound.args, **bound.kwargs)
  File "C:\Users\Alessandro\Documents\UCL-projects\napari-plugins\cellfinder-napari\cellfinder_napari\transform.py", line 72, in transform
    transformed_cells = transform_points_to_atlas_space(
  File "C:\Users\Alessandro\anaconda3\envs\brainglobe\lib\site-packages\cellfinder\analyse\analyse.py", line 230, in transform_points_to_atlas_space
    return transform_points_downsampled_to_atlas_space(
  File "C:\Users\Alessandro\anaconda3\envs\brainglobe\lib\site-packages\cellfinder\analyse\analyse.py", line 265, in transform_points_downsampled_to_atlas_space
    deformation_field = tifffile.imread(deformation_field_path)
  File "C:\Users\Alessandro\anaconda3\envs\brainglobe\lib\site-packages\tifffile\tifffile.py", line 1040, in imread
    with TiffFile(
  File "C:\Users\Alessandro\anaconda3\envs\brainglobe\lib\site-packages\tifffile\tifffile.py", line 3933, in __init__
    fh = FileHandle(file, mode=mode, name=name, offset=offset, size=size)
  File "C:\Users\Alessandro\anaconda3\envs\brainglobe\lib\site-packages\tifffile\tifffile.py", line 13635, in __init__
    self.open()
  File "C:\Users\Alessandro\anaconda3\envs\brainglobe\lib\site-packages\tifffile\tifffile.py", line 13650, in open
    self._fh = open(self._file, self._mode)  # type: ignore
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\dstansby\\brainreg-napari-output\\deformation_field_0.tiff'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\Alessandro\anaconda3\envs\brainglobe\lib\site-packages\magicgui\widgets\bases\_value_widget.py", line 65, in _on_value_change
    self.changed.emit(value)
  File "src\psygnal\_signal.py", line 905, in emit
  File "src\psygnal\_signal.py", line 952, in _run_emit_loop
psygnal._signal.EmitLoopError: calling <psygnal._weak_callback._StrongFunction object at 0x0000025851D9B5C0> with args=(False,) caused FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\dstansby\\brainreg-napari-output\\deformation_field_0.tiff'.
```
I suspect the hard-coded path here has something to do with it... 
{" "registration_output_folder": "/Users/dstansby/brainreg-napari-output"}
... but how did the CI pass all that time ago? 🤔